File: FileFinder.UI.Angular7/DocFinderApp/src/app/models/FileFindByTextResponse.ts
export interface IFileFindByTextResponse
{
    id : string;
    highlights : Array<string>
    title : string
    imageUrl : string
} 

export class FileFindByTextResponse implements IFileFindByTextResponse
{
    imageUrl: string;
    id : string;
    highlights : Array<string>
    title : string
}   

File: FileFinder.AppServices/Files/Queries/GetFileByText/FileFindByTextResponse.cs
using FileFinder.Domain.Entities;
using System;
using System.Collections.Generic;
using System.Text;

namespace FileFinder.AppServices.Files.Queries.GetFileByText
{
    public class FileFindByTextResponse
    {
        public Guid Id { get; set; }
        public IEnumerable<string> Highlights { get; set; }
        public string Title { get; set; }
        public string ImageUrl { get; set; }
    }
}
